File: 2026 05월/5월 3주차/breadbirds/03/[PGS - Lv1] 문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        int cntP = 0;
        int cntY = 0;
        for(int i = 0; i < s.length(); i++){
            if(s.charAt(i) == 'p' || s.charAt(i) == 'P'){
                cntP ++;
            }
            if(s.charAt(i) == 'y' || s.charAt(i) == 'Y'){
                cntY ++;
            }
        }
        if(cntP == cntY){
            answer = true;
        }
        else{
            answer = false;
        }

        // [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
        System.out.println("Hello Java");

        return answer;
    }
}

File: JeongEon8/02/[PGS - Lv1] 문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    int p = 0, y = 0;
    
    for(int i = 0; i < s.size(); ++i) {
        if(s[i] == 'p' || s[i] == 'P') p++;
        else if(s[i] == 'y' || s[i] == 'Y') y++;
    }
    if(p == 0 && y == 0) return true;
    else if(p == y) return true;
    else return false;
}
